pkg/apiserver/registry/inventory/virtualmachine/rest.go

```diff
@@ -85,73 +100,41 @@ func (r *REST) Get(ctx context.Context, name string, _ *metav1.GetOptions) (runt
 	return vm, nil
 }
 
+// List only supports three types of input options:
+//  1. All Namespaces.
+//  2. Label selector with only the specific Namespace, the only valid label selectors are
```

Comment:
I wont add each label and field on the comment. If subjected to change and we dont want to keep updating it.. Move it close to the code which uses it

Reply:
Sure